umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso_shared/use_constraints.clj

```diff
@@ -93,23 +101,44 @@
                            ;; entire matching string is returned and if there is a group in the regular
                            ;; expression, the first group of the matching string is returned.
                            (if (string? match) match (second match))))]
-      (str/join matches))))
+      (string/join matches))))
+
+(defn parse-ac
+  "Parse the passed in ISO XML access constraints description and value using the
+   passed in xpath and regular expression in this function."
```

Comment:
whitespace nitpick, style guide shows multi-line docstrings should start under the quote

Reply:
I have update the docstring.